nautobot/dcim/models/device_component_templates.py

```diff
@@ -465,54 +465,25 @@ def clean(self):
 
 
 @extras_features("custom_validators")
-class ModuleBayTemplate(BaseModel, ChangeLoggedModel, CustomFieldModel, RelationshipModel):
+class ModuleBayTemplate(ModularComponentTemplateModel):
     """Template for a slot in a Device or Module which can contain Modules."""
 
-    device_type = models.ForeignKey(
-        to="dcim.DeviceType",
-        on_delete=models.CASCADE,
-        related_name="module_bay_templates",
-        blank=True,
-        null=True,
-    )
-    module_type = models.ForeignKey(
-        to="dcim.ModuleType",
-        on_delete=models.CASCADE,
-        related_name="module_bay_templates",
-        blank=True,
-        null=True,
-    )
-    _position = NaturalOrderingField(target_field="position", max_length=CHARFIELD_MAX_LENGTH, blank=True)
     position = models.CharField(
         max_length=CHARFIELD_MAX_LENGTH,
-        blank=False,
-        null=False,
+        blank=True,
```

Comment:
Does this becoming blankable make #5779 more difficult to implement? Would that be easier if we keep this as a required field?

Reply:
Doing this for compatibility with devicetype library and a potential use case where you have nested modules but you want to use the position from the grandparent to template out your interface name and not the direct parent module bay